pkg/meta/base.go

```diff
@@ -935,9 +935,26 @@ func (m *baseMeta) Close(ctx Context, inode Ino) syscall.Errno {
 
 func (m *baseMeta) Readdir(ctx Context, inode Ino, plus uint8, entries *[]*Entry) syscall.Errno {
 	inode = m.checkRoot(inode)
-	var attr Attr
-	if err := m.GetAttr(ctx, inode, &attr); err != 0 {
-		return err
+	attr := &Attr{}
+	if m.conf.OpenCache == 0 || !m.of.OpenCheck(inode, attr) {
+		err := m.GetAttr(ctx, inode, attr)
+		if err != 0 {
+			return err
+		}
+		if m.conf.OpenCache > 0 {
```

Comment:
we should cache the result even OpenCache is zero (timeout)

Reply:
Ok, changed.